### PR TITLE
Add stable vector container

### PR DESCRIPTION
**Pull request by GraDKh**
Add StableVec structure

### CODE REVIEW

**Review by ufoscout — APPROVED** [review]

File: ic-stable-structures/src/storage/structures/vec.rs [hunk 1]
```diff
@@ -0,0 +1,182 @@
+use std::collections::BTreeMap;
+
+use ic_exports::{
+    ic_kit::ic,
+    stable_structures::{memory_manager::MemoryId, BoundedStorable, Vec},
+    Principal,
+};
```
Comment (F3kilo) [inline ic-stable-structures/src/storage/structures/vec.rs:7]: In our code style guide we decided to split such imports on module basis:
https://infinityswap.atlassian.net/wiki/spaces/CPROD/pages/24444929/Rust+Conventions#Import--/-use

**Review by F3kilo — APPROVED** [review]
Great work!
Just one code style suggestion.

File: ic-stable-structures/src/storage/structures/vec.rs (hunk 1 above)
Comment (GraDKh) [reply ic-stable-structures/src/storage/structures/vec.rs]: Thanks, fixed. For some reason `cargo +nightly fmt` didn't change that

**Comment by itsyaasir** [comment]
@GraDKh Can we add small documentation for the structure added ?

**Pull request closed without merging**